src/topography.f90

```diff
@@ -443,6 +444,29 @@ subroutine topography_min_max_depth(this, vgrid, level)
 
   end subroutine topography_min_max_depth
 
+  !-------------------------------------------------------------------------
+  subroutine topography_fill_fraction(this, sea_area_fraction)
+    class(topography_t), intent(inout) :: this
+    real(real32), intent(in) :: sea_area_fraction
+
+    write(output_unit,'(a,f7.2)') "Filling cells that have a sea area fraction smaller than ", sea_area_fraction
+
+    if (any(this%frac < sea_area_fraction)) then
+      where (this%frac < sea_area_fraction)
+        this%depth = 0.0
+        this%frac = 0.0
+      end where
+
+      if (this%lakes_removed == 'yes') then
+        ! We might have created new lakes, so rerun deseas
```

Comment:
clearer to say `! We might have created new lakes. To fix, rerun deseas.`

Reply:
I changed the comment to `! We might have created new lakes, so update the corresponding attribute`, as this comment is intended to explain why we are changing the attribute.